### PR TITLE
Add Concurrency

### PR DESCRIPTION
**Pull request by petrusck**
Hi!

Here the Concurrency framework is being added to the project. Now, in case of having multiple string catalogs, they will be checked in parallel.

The changes have been tested using the [compete concurrency check](https://www.swift.org/documentation/concurrency/) provided with the Swift Tools version 5.10 (`.enableExperimentalFeature("StrictConcurrency")`), to avoid including data-race safety issues and to assure the code is Swift 6 compatible.

Cheers!!

### CODE REVIEW

File: Sources/SwiftPolyglot/AsyncParsableCommand.swift [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import ArgumentParser
+
+protocol AsyncParsableCommand: ParsableCommand {
```
Comment (roddymunro) [inline Sources/SwiftPolyglot/AsyncParsableCommand.swift]: Doesn't this already exist? https://apple.github.io/swift-argument-parser/documentation/argumentparser/asyncparsablecommand/

**Review by roddymunro — COMMENTED** [review]
LGTM - just one question!

File: Sources/SwiftPolyglot/AsyncParsableCommand.swift (hunk 1 above)
Comment (pedrusck) [reply Sources/SwiftPolyglot/AsyncParsableCommand.swift]: Nice!! I didn't see it!! This simplifies a lot

The custom code is now being removed

**Comment by petrusck** [comment]
> LGTM - just one question!

Nice catch!! Fixed!!

**Comment by petrusck** [comment]
I also fixed the message shown with every missing translation (aka `MissingTranslation` structure's `message` property)

**Review by roddymunro — APPROVED** [review]

**Pull request merged**